tests/shared/src/test/scala/scala/meta/tests/tokenizers/TokenizerSuite.scala

```diff
@@ -2102,5 +2102,37 @@ class TokenizerSuite extends BaseTokenizerSuite {
                     |""".stripMargin.nl2lf
     assertTokenizedAsStructureLines(code, struct)
   }
+  test("broken-interpolator") {
+    val code = """|
+                  |s'''|Available languages (default = $):
+                  |verilog - Verilog or SystemVerilog dialects
+                  |vhdl    - VHDL dialects
+                  |
+                  |Available Verilog/SystemVerilog dialects (default = $defaultDialect):
+                  |v2001   - Verilog 2001
+                  |sv2005  - 
+                  |'''.stripMargin
+                  |""".stripMargin.replace("'''", "\"\"\"")
+
+    val struct = """|BOF [0..0)
+                    |LF [0..1)
+                    |Interpolation.Id(s) [1..2)
+                    |Interpolation.Start(''') [2..5)
+                    |Interpolation.Part(|Available languages (default = ) [5..37)
+                    |Interpolation.SpliceStart [37..38)
+                    |Invalid(Not one of: `$'_, `$$', `$'ident, `$'this, `$'BlockExpr) [38..38)
```

Comment:
I changed it a bit and I think it's currently the best option. Splice starts, but is broken, so we continue with normal string part and an invalid token.